app/views/shared/_personal_key.html.erb

```diff
@@ -23,7 +23,7 @@
 <br>
 <br>
 <div class="sm-col sm-col-2 padding-right-2">
-  <%= image_tag(asset_url('alert/icon-lock-alert-important.svg'), alt: '', size: '80') %>
+  <%= image_tag(asset_url('alert/icon-lock-alert-important.svg'), alt: 'Lock Icon with red exclamation point', size: '80') %>
```

Comment:
We should make sure this is translated based on the current locale.

Reply:
I think imma change this back. I think this icon may count as a decorative image, so probably doesn't need an alt. https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Ah, yeah, good point. That makes sense. 👍